spec/requests/page_views_spec.rb

```diff
@@ -4,6 +4,13 @@
   let(:user) { create(:user, :trusted) }
   let(:article) { create(:article) }
 
+  before do
+    # rubocop:disable RSpec/AnyInstance
+    allow_any_instance_of(PageViewsController).to receive(:skip_page_view_update?).and_return(false)
```

Comment:
If there is a better way to do this without using any_instance I am all ears but this is the best way I know how and in this case I dont see a lot of harm in using any_instance

Reply:
I oddly feel like this works well for this given scenario 🙈

Reply:
This might be overkill but you could extract this `rand()` usage into a module and call that. Then you can just stub the module without using `allow_any_instance_of`.